### DISCUSSION TITLE
Authelia and Remotely

### DISCUSSION THREAD

**Discussion by Yel-vnk** [authelia/authelia#5091] [Q&A]
### Version

v4.37.5

### Deployment Method

Docker

### Reverse Proxy

NGINX Proxy Manager

### Description

Hi everyone ,
i need your help please , I have a issue with remotely and authelia
My setup is :

Client => Ngnix Reverse Proxy => Authelia => remotely

When i try to acces remotely a get authelia to authenticate nice
but : when i try to connect a remote agent it can't see remtely web site

is anyone make remotely work with authelia ?

Thank you for your support ,

### Reproduction

Installing remotely and use authelia to acces it 

### Logs (Authelia)

```shell
No Avalible
```

### Pre-Submission Checklist

- [X] I agree to follow the [Code of Conduct](http://www.authelia.com/code-of-conduct)
- [X] This is a bug report and not a support request
- [X] I have read the security policy and this bug report is not a security issue or security related issue
- [X] I have either included the complete configuration file or I am sure it's unrelated to the configuration
- [X] I have provided all of the required information in full with the only alteration being reasonable sanitization in accordance with the [Troubleshooting Sanitization](https://www.authelia.com/r/sanitize) reference guide
- [X] I have checked for related proxy or application logs and included them if available
- [X] I have checked for related issues and checked the documentation

**Comment by james-d-elliott** [comment]
Since no one has had this issue that's seen it maybe you can provide some more information to help diagnose it, i.e. logs (debug or trace) from both authelia and the app in question.

**Comment by pr0927** [comment]
I am having this issue as well. Using Remotely from TrueNAS SCALE, via TrueCharts app.

Traefik is my reverse proxy, Authelia is my middleware. I had to disable Cloudflare proxy for the remote.mydomain.tld CNAME entry for my Remotely instance, else no settings would save.

Then I couldn't get a session ID to remote connect like OP mentioned, with Authelia.

**Comment by james-d-elliott** [comment]
As no additional debug information has been supplied when requested [here](https://github.com/authelia/authelia/discussions/5091#discussioncomment-5499259) I'm closing this.
